tests/FakeItEasy.IntegrationTests/TypeCatalogueTests.cs

```diff
@@ -131,6 +131,23 @@ public void Should_not_be_able_to_get_types_from_assembly_that_does_not_referenc
             catalogue.GetAvailableTypes().Should().NotContain(typeof(string));
         }
 
+        [Fact]
+        public void Should_warn_if_some_types_cannot_be_loaded_from_external_assembly()
+        {
+            // Arrange
+            var catalogue = new TypeCatalogue();
+
+            // Act
+            string output = CaptureConsoleOutput(
+                () => catalogue.Load(new[] { this.externalAssemblyGenerator.AssemblyOriginalPath }));
+
+            // Assert
+            output.Should().Match(@"*Warning: FakeItEasy failed to get some types from assembly 'FakeItEasy.ExtensionPoints.External, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' while scanning for extension points. Some IArgumentValueFormatters, IDummyFactories, and IFakeOptionsBuilders in that assembly might not be available.
+  System.Reflection.ReflectionTypeLoadException: *.
+  1 type(s) were not loaded for the following reasons:
+   - System.IO.FileNotFoundException: *");

```

Comment:
Using wildcards instead of the actual exception messages, so that it still works on non-English systems.